test/tribes.js

```diff
@@ -338,5 +341,27 @@ test('You can add multiple keys to a group and switch between them for writing',
     'picking second key worked'
   )
 
-  tearDown()
+  await tearDown()
+})
+
+test('You can exclude info from a group', async (t) => {
+  setup()
+
+  await sbot.box2.addGroupInfo(groupId, { key: testkey, root: testRoot })
+
+  await sbot.box2.excludeGroupInfo(groupId, null)
+
+  const groupInfo = await sbot.box2.getGroupInfo(groupId)
+
+  t.deepEquals(
+    groupInfo,
+    { excluded: true },
+    'excluding group info just leaves excluded: true'
+  )
+
+  const list = await pull(sbot.box2.listGroupIds(), pull.collectAsPromise())
+
+  t.deepEquals(list, [groupId], 'group is still in list after exclusion')
```

Comment:
same question as other PR - is this here still so you can know you were excluded?

Reply:
yes that's part of the anti-gaslighting requirement: people should know that they got removed from the group, without wondering if they were in it in the first place